.gitignore

```
@@ -9,3 +9,9 @@
 
 # misc
 .DS_Store
+
+# cypress
+cypress.env.json
+cypress/downloads/
+cypress/screenshots/
+cypress/videos/
```

```suggestion
cypress/videos/

```

É uma boa prática deixar uma linha em branco ao final de cada arquivo.

Dessa forma, ferramentas como GitHub, por exemplo, não exibem o ícone de _No newline at end of file_.